pkgs/applications/networking/cluster/calico/default.nix

```diff
@@ -0,0 +1,81 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+builtins.mapAttrs (pname: { doCheck ? true, mainProgram ? pname, subPackages }: buildGoModule rec {
+  inherit pname;
+  version = "3.24.5";
+
+  src = fetchFromGitHub {
+    owner = "projectcalico";
+    repo = pname;
```

Comment:
These sources cannot be downloaded because repository was unified at https://github.com/projectcalico/calico. `nixpkgs-review` fails for me because of that.

Reply:
yeah, it was a refactor bug, back from when I only had the single derivation named `calico`; I will change the commit and pr title to match.

love how #170350 masks this